content-repo/extra-docs/releases/20.5.2.md

```diff
@@ -0,0 +1,271 @@
+## Cortex XSOAR Release Notes for version 20.5.2 (54345)
+##### Published on 26 May 2020
+
+### Breaking Changes
+- __TrueSTAR__  
+Changed several arguments and outputs in **trustar-get-phishing-submissions** and **trustar-get-phishing-indicators** commands. See the **Integrations** section for full details.
+
+**End Of Life Notice**: *Palo Alto Networks Cortex* Integration will reach end of life on May 31st. This is due to changes in the Cortex Data Lake move to a new version 2.0 API. Please make sure to use the *Cortex Data Lake* Integration instead.
+
+### Integrations
+
+#### 5 New Integrations
+- __IllusiveNetworks__  
+The Illusive Attack Management API allows customers to retrieve detected incidents with a forensics timeline, attack surface insights, collect forensics on-demand, and manage a variety of operations with regard to deceptive entities, deception policies, and more.
+- __Bastille Networks__  
+RF monitoring for wireless intrusion detection and policy enforcement. Visit https://www.bastille.net for details.
+- __Logz.io__  
+Fetch & remediate security incidents identified by Logz.io Cloud SIEM.
+- __Digital Guardian__  
+Use Digital Guardian Integration to fetch incidents and to programmatically add or remove entries from watchlists and component lists.
+- __ThreatMiner__  
+Data Mining for Threat Intelligence.
+
+#### 26 Improved Integrations
+- __TruSTAR__  
+  - *Breaking changes:*
+    - *normalized_triage_score* argument replaced by *priority_event_score* in *trustar-get-phishing-submissions* and *trustar-get-phishing-indicators* command.
+    - Updated context outputs on *trustar-get-phishing-submissions* and *trustar-get-phishing-indicators*.  
+    - *normalized_source_score* argument replaced by *normalized_indicator_score* in *trustar-get-phishing-indicators* command.
+  - *Non Breaking changes:*
+    - Fixed *from_time* description on *trustar-get-phishing-indicators* and *trustar-get-phishing-submissions* to '24 hours ago'.
+    - Added -1 to list of default values in *priority_event_score* on *trustar-get-phishing-submissions*.
+    - Added -1 to list of default values in *priority_event_score* and *normalized_indicator_score* on *trustar-get-phishing-indicators*.
+- __Microsoft Graph Security__  
+Added **fetch-incidents** functionality.
```

Comment:
lets be consistent with the fetch incidents, here it says `incidents` and formatted in bold while 2 lines below it says `incident` and in italic